### DISCUSSION TITLE
Possibility to restore different snapshots

### DISCUSSION THREAD

**Discussion by shrippen** [zbm-dev/zfsbootmenu#632] [Feature Requests]
Hi,

zbm has the ability to list all my snapshots and let me select one to rollback to. Unfortunately I seem to be locked to what snapshots are shown in the list.

If I want to rollback my installation i have to rollback 5 different zvols and it would be a lot fast if I could also do the other ones via the gui.

So my request is to have to ability to also show snapshots for other zvols.

**Comment by ahesford** [comment]
This has been discussed before, and we do not want to go down the path of integrating a general-purpose snapshot management tool into ZFSBootMenu. We only allow rollbacks of the root because that must happen before the root filesystem is mounted. For everything else, you can use whatever tools you like in your booted system, or you can jump into a chroot and work from there in a pre-boot environment.

You may have some success with `zsnapshots` in the recovery shell, but that's about as far as we'll go.
